pkg/cmd/roachtest/tests/admission_control_latency.go

```diff
@@ -785,9 +798,9 @@ func (v variations) initWorkload(ctx context.Context, l *logger.Logger) {
 // Don't run a workload against the node we're going to shut down.
 func (v variations) runWorkload(ctx context.Context, duration time.Duration, maxRate int) error {
 	runCmd := fmt.Sprintf(
-		"./cockroach workload run kv --duration=%s --max-rate=%d --tolerate-errors --max-block-bytes=%d --read-percent=50 --follower-read-percent=50 --concurrency=500 --operation-receiver=%s:%d {pgurl:1-%d}",
-		duration, maxRate, v.maxBlockBytes, v.roachtestAddr, v.histogramPort, v.stableNodes)
-	return v.cluster.RunE(ctx, option.WithNodes(v.cluster.Node(v.workloadNode)), runCmd)
+		"./cockroach workload run kv --duration=%s --max-rate=%d --tolerate-errors --max-block-bytes=%d --read-percent=50 --follower-read-percent=50 --concurrency=500 --operation-receiver=%s:%d {pgurl%s}",
+		duration, maxRate, v.maxBlockBytes, v.roachtestAddr, v.histogramPort, v.stableNodes())
```

Comment:
Does `option.NodeListOption` get evaluated to a pgurl format when calling `String()`?

Reply:
Yes - see https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/option/node_list_option.go#L91-L125